app/src/main/java/org/xbmc/kore/ui/generic/NavigationDrawerFragment.java

```diff
@@ -81,9 +82,11 @@
             ACTIVITY_TVSHOWS = 3,
             ACTIVITY_MUSIC = 4,
             ACTIVITY_PVR = 5,
-            ACTIVITY_FILES = 6,
-            ACTIVITY_ADDONS = 7,
-            ACTIVITY_SETTINGS = 8;
+            ACTIVITY_FAVOURITES = 6,
+            ACTIVITY_FILES = 7,
+            ACTIVITY_ADDONS = 8,
+            ACTIVITY_SETTINGS = 9;
+
```

Comment:
Slight bug over here. Actually it's just proof that this part of the code isn't very robust, but it is what it is right now, so we'll have to live with it...

You can't change the ids of the existing constants because these are already stored in the user's preferences (to show/hide the menu items). Meaning that what is stored in preferences is this actual number, so any change here will collide with that.
You'll have to keep the current activities as is and add ACTIVITY_FAVORITES as the 9. A side effect is that when the user receives the upgraded version, the favorites link won't be shown on the menu, because it won't be stored in the preferences... He will have to manually go into settings->side menu items and check the favorites. 

Alternatively (and ideally) we need to, ONCE, add this entry to the preference, which can be done on the remote activity. This is similar to the code that was removed in 39ee0b6634bffe5451912f63f67cf978632f92d7 (this was the check to introduce the PVR section, and was removed in the next version as it should no longer be necessary).
If this is not clear to you or you're not confortable with it, don't mind it, just don't change the previous IDs, add to them.